Sources/iron/object/BoneAnimation.hx

```diff
@@ -93,7 +93,14 @@ class BoneAnimation extends Animation {
 		}
 		ar.push(o);
 	}
-
+	
+	public function removeBoneChild(bone: String, o: Object) {
+		if (boneChildren != null){
+		var ar: Array<Object> = boneChildren.get(bone);
+		if (ar.indexOf(o) != -1) ar.remove(o);
```

Comment:
You probably don't need to check if `o` exists in the array, according to the Haxe documentation it seems that this is done already in `ar.remove(o)`: https://api.haxe.org/Array.html#remove

Reply:
cool! lets erase that then